config/gulp/sass.js

```diff
@@ -33,6 +36,43 @@ gulp.task('scss-lint', function (done) {
 
 });
 
+gulp.task('create-tmp-docs', function (done) {
+  return gulp.src('docs/doc_assets/css/**/*.scss')
+    .pipe(replace(/---/g, ''))
+    .pipe(gulp.dest('tmpdocs'));
+});
+
+gulp.task('lint-tmp-docs-scss', function (done) {
```

Comment:
On further exception, I've also noticed that all linting line numbers are now off by three because of the empty lines at the top of the file without the `---` triple dashes.